src/lib/configs/project/dockerCompose.ts

```diff
@@ -45,11 +59,68 @@ import {
   type Migrations,
 } from "../initConfig.js";
 
+import type { FluenceConfigReadonly } from "./fluence.js";
 import {
+  commonNoxConfig,
+  type NoxConfigYAML,
   initNewReadonlyProviderConfig,
   type ProviderConfigReadonly,
 } from "./provider.js";
 
+const NOX_IPFS_MULTIADDR = `/dns4/${IPFS_CONTAINER_NAME}/tcp/${IPFS_PORT}`;
+
+function getIsLocal() {
+  if (envConfig?.fluenceEnv === undefined) {
+    return true;
+  }
```

Comment:
Why do you interpret empty `env.yml` to mean `env: local`?

Reply:
That's a good catch, I actually don't remember why I did this to be honest. Probly wanted to try something out quickly and forgot to fix it. Anyway I am thinking that with multiple porvider.yaml in one dir we would have to set `env` right in provider.yaml itself, though I previously thought this would be unnecessary. So will remove this part and get env from provider config

Reply:
Hm, but `provider.yml` won't set Env for the Developer, right?

ie, imagine a project with _just_ `fluence.yml`, where would you store the selected env?

or maybe I understood your thinking wrong :)

Reply:
provider.yaml will not set env for developer. It will only set env for itself so when provider generates stuff from it - it's generated for the correct env.

developer will still use env.yaml under the hood as before when working with the project. and for local development `env: local` will be preset in provider.yaml